src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,19 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      <p>
+        {' Posted by  '}
+
+        <UserInfo name={post.user.name} email={post.user.email} />
```

Comment:
```suggestion
        user && <UserInfo name={user.name} email={user.email} />
```
Consider checking if the user exists